static/app/views/eventsV2/table/tableView.tsx

```diff
@@ -200,22 +200,22 @@ class TableView extends React.Component<TableViewProps> {
     }
     const currentSort = eventView.sortForField(field, tableMeta);
     const canSort = isFieldSortable(field, tableMeta);
+    const title = isEquationAlias(column.name)
+      ? eventView.getEquations()[getEquationAliasIndex(column.name)]
+      : column.name;
+    const shouldTruncate = title.length > 60;
 
     const sortLink = (
       <SortLink
         align={align}
-        title={column.name}
+        title={title.substring(0, 60) + (shouldTruncate ? '...' : '')}
```

Comment:
Can you just wrap this in a truncate instead maybe? Handrolling substrings feels a bit meh.